src/intTest/kotlin/no/nav/personbruker/dittnav/eventaggregator/done/CachedDoneEventConsumerTest.kt

```diff
@@ -24,13 +24,13 @@ class CachedDoneEventConsumerTest {
     private val doneRepository = DoneRepository(database)
     private val eventConsumer = CachedDoneEventConsumer(doneRepository)
 
-    private val produsent = "dummyProducer"
+    private val systembruker = "dummyProducer"
```

Comment:
Burde det også stå "dummySystembruker"? Eller blir det for pirkete? 😜

Reply:
Nei jeg vurderte det ;) Lurte bare om det da ble tydeligere at systembruker er produsent av eventet, om du forstår.. Men mulig det beste er å kalle det dummySystembruker, ja :)

Reply:
Ja, da skiller vi hvertfall systembruker helt fra produsent, men jeg har ikke noen sterke meninger om det 😉

Reply:
Jeg tenker også at det kan være fint å kalle det for dummySystembruker, så er vi helt tydelige 😃

Reply:
Endret disse strengene nå.